assembl/templates/views/messageSend.tmpl

```diff
@@ -1,9 +1,10 @@
 <% if (canPost) { %>
-    <% if ( !reply_message_id ) { %>
```

Comment:
make sure this still works when you reply on a single-column thread, because it seems that the same view is used